index.html

```diff
@@ -22,17 +22,19 @@ <h1 class="name">
   <section id="profile-picture">
     <img src="http://placehold.it/960X450/000000"/>
     </section>
+  <script id="article-template" type="text/x-handlebars-template">
```

Comment:
Recommend moving this script tag into the head section for easier readability. You should keep the projects section in this spot, just move the template code.